README.md

```diff
@@ -1,15 +1,37 @@
 # bfint
 
-BrainFuck interpreter implemented in Rust.
+This is a BrainFuck interpreter implemented in Rust. BrainFuck is an esoteric programming language known for its minimalism and extreme simplicity.
 
+## Usage
 
-## Roadmap
+The bfint-cli run command supports two options:
+
+- `[ARG]`: Positional an argument with BrainFuck code.
+- `--file <FILE>`: Specifies the file containing the BrainFuck code to run.
```

Comment:
nit
This can be done like in python and many others interpreted languages
-c argument is code
[args] positional argument is file with code

But here vice versa

Reply:
Okay agree with u, I will change my code